pkg/controller/core/oam.go

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
pkg/controller/setup.go  is better?

pkg/controller/group_x/traits/xxx_trait_ctrl
pkg/controller/ext/workload/xxx_workload_ctrl


They are all called by this setup function

Reply:
I like the name. I am not sure what "group-x" means. The directory names under pkg/controller are following https://github.com/oam-dev/spec/blob/master/3.workload.md#core-workload